tests/framework/console/MarkdownTest.php

```
@@ -0,0 +1,69 @@
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->destroyApplication();
+        $this->parser = new Markdown();
+    }
+
+    public function testRenderStrong(): void
+    {
+        $result = $this->parser->parse('**bold**');
+        $stripped = Console::stripAnsiFormat($result);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\yii\helpers\Console' in method 'testRenderStrong'. (undefined)

(StaticAccess)